README.md

```diff
@@ -79,7 +79,7 @@ Imagine that you take the code from this question, save it to a file called `foo
 
 What would be the output? Explain your answer.
 
-<!-- Replace this comment with your answer -->
+it will display "sushi is tasty". because 2 is greater than 0 but the first if statement still evaluates to false.
```

Comment:
Accidentally missed this. Unfortunately, this answer is not correct. The reason is that `x` is not defined - when our program tries to run through this line, it gives us an error.